app/routes.py

```diff
@@ -55,21 +54,21 @@ def registration():
         return redirect(url_for('/'))
     return render_template('account-creation.html', form=account_creation)
 
-#This route points to a button which will send an email.
-@app.route('/send_email_button', methods=['GET', 'POST'])
-def sending_emails():
-    send = Email()
-    send.send_email()
-    return render_template('main.html')
-
-@app.route('/see_editor_picks', methods=['GET', 'POST'])
-def retrieve_editor_picks():
-    return render_template('editor-picks.html')
+@app.route('/user_profile', methods=['GET', 'POST'])
+def retrieve_user_profile():
+    #review_display = ReviewDisplay()
+    #review_display.display_database_reviews()
+    return render_template('User Profile.html')
 
-@app.route('/return_to_main', methods=['GET', 'POST'])
+@app.route('/return_to_menu', methods=['GET', 'POST'])
 def return_to_main_menu():
     return render_template('main.html')
```

Comment:
Not necessary. Look at using an if statement and the redirect method from Flask.